src/StochasticNoiseSimulator.cpp

```diff
@@ -143,24 +132,39 @@ std::map<std::string, double> StochasticNoiseSimulator::StochSimulate() {
     }
     const auto t2_stoch = std::chrono::steady_clock::now();
     stoch_run_time      = std::chrono::duration<float>(t2_stoch - t1_stoch).count();
+
+    for (unsigned long j = 0; j < max_instances; j++) {
+        for (const auto& classical_measurements_map: classical_measurements_maps[j]) {
```

Comment:
consider using structured bindings here to make the intent a bit clearer, i.e., 
`const auto& [something, else]: classicalMeasurementMaps[j]`

Even better: use in conjunction with a range-based for loops instead of the iteration over `j`

Reply:
I am not sure if this would work here. Instead, I added a comment to make the intent a bit clearer.

Reply:
I'll take a look myself and if it works, I'll commit the change.